context/VendorContext.js

```diff
@@ -0,0 +1,89 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import venuePic1 from "../public/images/venue1.jpg";
+import venuePic2 from "../public/images/venue2.jpeg";
+import venuePic3 from "../public/images/venue3.jpeg";
+import venuePic4 from "../public/images/venue4.jpeg";
+import photographyPic1 from "../public/images/photography1.jpeg";
+import photographyPic2 from "../public/images/photography2.jpeg";
+import photographyPic3 from "../public/images/photography3.jpeg";
+import photographyPic4 from "../public/images/photography4.jpeg";
+import photographyPic5 from "../public/images/photography5.jpeg";
```

Comment:
just a small note: since you are importing the images here, next.js will put them in your bundle anyway. Therefore you don't need to put them in the public folder.